### PR TITLE
Peddy backup

### PR DESCRIPTION
**Pull request by sima-r**
Added Peddy check module and subworkflow to the pipeline. 
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test2samples,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/peddy_check.nf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+workflow PEDDY_CHECK {
+    take:
+        vcf                // channel: [ val(meta), path(vcf), path(vcf_index) ]
+        ped                
+        
+    main:
+        ch_versions = Channel.empty()
+
+        PEDDY(vcf, ped)
+        ch_versions = ch_versions.mix(PEDDY.out.versions)
```
Comment (jemten) [inline subworkflows/local/peddy_check.nf:16]:
```suggestion
        ch_versions = ch_versions.mix(PEDDY.out.versions.first())
```

File: subworkflows/local/peddy_check.nf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        
+    emit:
+        ped                    = PEDDY.out.ped
+        csv                    = PEDDY.out.csv
+        versions               = ch_versions.ifEmpty(null)      // channel: [ versions.yml ]
```
Comment (jemten) [inline subworkflows/local/peddy_check.nf:21]:
```suggestion
        ped                    = PEDDY.out.ped
        csv                    = PEDDY.out.csv
        versions               = ch_versions      // channel: [ versions.yml ]
```

File: subworkflows/local/peddy_check.nf [hunk 3]
```diff
@@ -0,0 +1,24 @@
+//
+// Peddy subworkflow to check sex and relatedness.
+//
+
+include { PEDDY } from '../../modules/nf-core/peddy/main'
+
+workflow PEDDY_CHECK {
+    take:
+        vcf                // channel: [ val(meta), path(vcf), path(vcf_index) ]
+        ped
```
Comment (jemten) [inline subworkflows/local/peddy_check.nf:10]: Could you describe the ped channel in the same way as done for the vcf.

File: subworkflows/local/peddy_check.nf (hunk 2 above)
Comment (jemten) [reply subworkflows/local/peddy_check.nf:21]: Could you add a channel description as done for the input vcf

**Review by jemten — COMMENTED** [review]
Some small comments. Otherwise it looks good

File: conf/modules/peddy_check.config [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
+        ext.when            = Conditional clause
+----------------------------------------------------------------------------------------
+*/
+
+//
+// Gens options
```
Comment (ramprasadn) [inline conf/modules/peddy_check.config:15]:
```suggestion
// Peddy options
```

**Review by ramprasadn — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**